core/src/main/java/org/bitcoinj/coinjoin/PoolStatusUpdate.java

```diff
@@ -13,10 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.bitcoinj.core;
+package org.bitcoinj.coinjoin;
 
-public class DarkSendQueue extends EmptyMessage {
```

Comment:
`DarkSendQueue` is removed since `dsq` is implemented by `CoinJoinQueue` now.

Reply:
I wonder why git thought this file was moved and changed rather than deleted.